src/biokbase/narrative/jobs/jobmanager.py

```diff
@@ -770,7 +771,7 @@ def list_jobs(self):
         """
         try:
             all_states = self.get_all_job_states(ignore_refresh_flag=True)
-            state_list = [s["jobState"] for s in all_states.values()]
+            state_list = [copy.deepcopy(s["jobState"]) for s in all_states.values()]
```

Comment:
Shouldn't need this anymore since we fixed the double- yet leaky-caching, but no biggie

Reply:
would you mind fixing it in the next PR?